docs/release-notes/v12.0.0.md

```diff
@@ -135,20 +135,24 @@ To further improve the automatic configuration of the Fast Data, from now on whe
 
 Every new projection created will automatically set the ingestion topic to the pattern `{tenantId}.{projectId}.{systemId}.{projectionId}.ingestion` and the pr-update topic with `{tenantId}.{projectId}.{systemId}.{projectionId}.pr-update`. These are editable and it is possible to use environment variables for the entire name or part of the name. This lets you differentiate topics for each environment leveraging on the usage of environment variables.
 
-:::info
+:::caution
 A migration will perform a check on the ingestion topic names of existing projections. If they share the same name among environments, this name will be used as the unique ingestion topic name. Otherwise, a public environment variable will be created with the values: its name will follow the pattern `FAST_DATA_{topicName}_INGESTION_TOPIC` and it will be also set in the projection configuration page. 
 
 The same approach is used for the pr-update topics, and the eventual public variable generated will be called `FAST_DATA_{topicName}_PR_UPDATE_TOPIC`.
+:::
 
 The environment variable(s) can be found in the _Public Variables_ section and they are editable. 
 
 An empty commit is required to save these modifications.
-:::
 
-:::info
 Because of these updates, the public variables of the invariant topic map (usually named `FAST_DATA_{realTimeUpdaterName}_INVARIANT_TOPIC_MAP`) are not used anymore by the Real-Time Updater services attached to a System of Record.
 
 These variables are still available as set as editable. They can be deleted after verifying that they are not used any longer.
+
+:::danger
+If you have enabled the emission of pr-updates in a Real-Time Updater/Projection Storer but not all projections should emit the events, make sure you clear the unused pr-update topic inputs inside the details page of each excluded projection.
+
+![Kafka Topics section inside Projection details page](./img/kafka_topics_projection_page.png)
```

Comment:
I think this should stay in the RN of the v12.0.3.

Reply:
if someone is using a non-stable version they will receive the fix with this version so I think the bugfix should remain, shouldn't it?

Reply:
Yeah, at the end we decided to rephrase it. 
Closing this conversation.